repairnator/repairnator-pipeline/src/main/resources/prophet4j/_BIN/README.md

```diff
@@ -0,0 +1,14 @@
+Prophet4J is inside the Coming lib, and is introduced to compute OverfittingScores for patches.
+
+This folder contains necessary resources for Prophet4J. There are two same folders in total, one is at _src/main/resources_ and the other one is at _src/test/resources_. I had not found one correct way to import resources files to make Prophet4J work smoothly as one part of Maven lib. In ideal cases, we do not need these two folders at all.
```

Comment:
do we need this hidden README file?

(avoid folder named `_BIN`)

Reply:
Yes, we do. It warns maintainers not to delete this folder casually.
(the naming of `_BIN` has been specified in the Coming lib, so it will become useless once gets renamed)